Release 2021-12-06

FSM1 opened the pull request:
- add template for pull requests (#1751) 
- Bump ssri from 6.0.1 to 6.0.2 (#1747) 
- Bump merge-deep from 3.0.2 to 3.0.3 (#1744) 
- Bump dns-packet from 1.3.1 to 1.3.4 (#1745) 
- Bump tmpl from 1.0.4 to 1.0.5 (#1741) 
- Bump path-parse from 1.0.6 to 1.0.7 (#1742) 
- Bump color-string from 1.5.3 to 1.6.0 (#1743) 
- Bump hosted-git-info from 2.8.8 to 2.8.9 (#1746) 
- bump cypress to v9.0 (#1749) 
- Bump url-parse from 1.4.7 to 1.5.3 (#1748) 
- Link sharing redesign (#1722) 
- Placeholder spacing in tag input (#1731) 
- Add subdirectory for pr template (#1756)
- add test for hotkey navigation in file preview (#1755)
- fix video preview overflow (#1752)
- Share files from browser support (#1736) 
- Revert "copy working" (#1759) 
- Translations update from Hosted Weblate (#1753) 
- [ImgBot] Optimize images (#1767) 
- Report modal styles and cut off fix (#1766) 
- Fix Mobile Nav Drawer blocks breadcrumb (#1768)
- secondary colors update (#1769) 
- File info modal CID and Key copy arrangements (#1773) 
- fix share transfer modal alignment (#1774) 
- move pr template to .github root dir (#1776) 
- Storage Maintenance Mode fix (#1777)
- resolve double tap (#1778) 
- Mobile sharing folder rename modal (#1779) 
- Rename with split extension (#1760) 
- Add ui test coverage for link creation / deletion (#1775)
- Translated using Weblate (French) (#1784) 
- Bump Cypress to 9.1 (#1785) 
- removing gaming dashboard from monorepo (#1790) 
- Fix 0 randomly appearing (#1789) 
- Fix typing issues that appeared with cypress v9.1 upgrade (#1791) 
- Edit PR template (#1798) 
- Heic loader (#1765) 
- change for create (#1797) 
- attempt fixing storybook build (#1801)
- add sorting header for mobile file browser (#1786)
- Add header content for selected (#1796)

asnaith committed:
add template for pull requests (#1751)

Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

dependabot[bot] committed:
Bump ssri from 6.0.1 to 6.0.2 (#1747)

Bumps [ssri](https://github.com/npm/ssri) from 6.0.1 to 6.0.2.
- [Release notes](https://github.com/npm/ssri/releases)
- [Changelog](https://github.com/npm/ssri/blob/v6.0.2/CHANGELOG.md)
- [Commits](https://github.com/npm/ssri/compare/v6.0.1...v6.0.2)

---
updated-dependencies:
- dependency-name: ssri
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

dependabot[bot] committed:
Bump merge-deep from 3.0.2 to 3.0.3 (#1744)

Bumps [merge-deep](https://github.com/jonschlinkert/merge-deep) from 3.0.2 to 3.0.3.
- [Release notes](https://github.com/jonschlinkert/merge-deep/releases)
- [Commits](https://github.com/jonschlinkert/merge-deep/compare/3.0.2...3.0.3)

---
updated-dependencies:
- dependency-name: merge-deep
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

dependabot[bot] committed:
Bump dns-packet from 1.3.1 to 1.3.4 (#1745)

Bumps [dns-packet](https://github.com/mafintosh/dns-packet) from 1.3.1 to 1.3.4.
- [Release notes](https://github.com/mafintosh/dns-packet/releases)
- [Changelog](https://github.com/mafintosh/dns-packet/blob/master/CHANGELOG.md)
- [Commits](https://github.com/mafintosh/dns-packet/compare/v1.3.1...v1.3.4)

---
updated-dependencies:
- dependency-name: dns-packet
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump tmpl from 1.0.4 to 1.0.5 (#1741)

Bumps [tmpl](https://github.com/daaku/nodejs-tmpl) from 1.0.4 to 1.0.5.
- [Release notes](https://github.com/daaku/nodejs-tmpl/releases)
- [Commits](https://github.com/daaku/nodejs-tmpl/commits/v1.0.5)

---
updated-dependencies:
- dependency-name: tmpl
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump path-parse from 1.0.6 to 1.0.7 (#1742)

Bumps [path-parse](https://github.com/jbgutierrez/path-parse) from 1.0.6 to 1.0.7.
- [Release notes](https://github.com/jbgutierrez/path-parse/releases)
- [Commits](https://github.com/jbgutierrez/path-parse/commits/v1.0.7)

---
updated-dependencies:
- dependency-name: path-parse
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump color-string from 1.5.3 to 1.6.0 (#1743)

Bumps [color-string](https://github.com/Qix-/color-string) from 1.5.3 to 1.6.0.
- [Release notes](https://github.com/Qix-/color-string/releases)
- [Changelog](https://github.com/Qix-/color-string/blob/master/CHANGELOG.md)
- [Commits](https://github.com/Qix-/color-string/commits/1.6.0)

---
updated-dependencies:
- dependency-name: color-string
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

dependabot[bot] committed:
Bump hosted-git-info from 2.8.8 to 2.8.9 (#1746)

Bumps [hosted-git-info](https://github.com/npm/hosted-git-info) from 2.8.8 to 2.8.9.
- [Release notes](https://github.com/npm/hosted-git-info/releases)
- [Changelog](https://github.com/npm/hosted-git-info/blob/v2.8.9/CHANGELOG.md)
- [Commits](https://github.com/npm/hosted-git-info/compare/v2.8.8...v2.8.9)

---
updated-dependencies:
- dependency-name: hosted-git-info
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

asnaith committed:
bump cypress to v9.0 (#1749)

Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

dependabot[bot] committed:
Bump url-parse from 1.4.7 to 1.5.3 (#1748)

Bumps [url-parse](https://github.com/unshiftio/url-parse) from 1.4.7 to 1.5.3.
- [Release notes](https://github.com/unshiftio/url-parse/releases)
- [Commits](https://github.com/unshiftio/url-parse/compare/1.4.7...1.5.3)

---
updated-dependencies:
- dependency-name: url-parse
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Tbaut committed:
Link sharing redesign (#1722)

* redesign

* copy buttons

* lingui extract

* dynamic dropdown

* copy working

* remove label

* styling suggestions

* lingui extract

Co-authored-by: GitHub Actions <actions@github.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>
Co-authored-by: Tanmoy Basak Anjan <tanmoy3399@gmail.com>

RyRy79261 committed:
Resolved (#1731)

Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>
Co-authored-by: Tanmoy Basak Anjan <tanmoy3399@gmail.com>

asnaith committed:
Add subdirectory for pr template (#1756)

asnaith committed:
add test for hotkey navigation in file preview (#1755)

FSM1 committed:
fix video preview overflow (#1752)

tanmoyAtb committed:
Share files from browser support (#1736)

* redesign

* copy buttons

* lingui extract

* dynamic dropdown

* copy working

* remove label

* styling suggestions

* lingui extract

* landing modal alignments

* lingui extract

* modal config

* lingui extract

* manage shared folder code refactor

* lingui extract

* modals progress working

* added posthog event

* lingui extract

* changing widths

* added width  breaks

* removed extra styled

* compile translations

* added status checks

* close instead of cancel

Co-authored-by: Thibaut Sardan <github@thib.top>
Co-authored-by: GitHub Actions <actions@github.com>
Co-authored-by: Thibaut Sardan <33178835+Tbaut@users.noreply.github.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

FSM1 committed:
Revert "copy working" (#1759)

This reverts commit 424f4278a558c64d6d9317693aa40a4ee477dd03.

Co-authored-by: Tanmoy Basak Anjan <tanmoy3399@gmail.com>

weblate committed:
Translations update from Hosted Weblate (#1753)

* Translated using Weblate (French)

Currently translated at 100.0% (304 of 304 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/fr/

* Translated using Weblate (French)

Currently translated at 100.0% (310 of 310 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/fr/

* Translated using Weblate (French)

Currently translated at 100.0% (317 of 317 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/fr/

Co-authored-by: J. Lavoie <j.lavoie@net-c.ca>

imgbot[bot] committed:
[ImgBot] Optimize images (#1767)

*Total -- 66.99kb -> 35.67kb (46.75%)

/packages/files-ui/cypress/fixtures/uploadedFiles/chainsafe.png -- 38.39kb -> 14.61kb (61.94%)
/packages/files-ui/src/Media/sharingExplainer/step2.png -- 9.24kb -> 4.60kb (50.19%)
/packages/files-ui/src/Media/sharingExplainer/step1.png -- 5.66kb -> 3.65kb (35.47%)
/packages/common-components/src/Icons/svgs/ethereum-logo.svg -- 0.78kb -> 0.62kb (20.88%)
/packages/files-ui/src/Media/sharingExplainer/step3.png -- 12.38kb -> 11.66kb (5.87%)
/packages/common-components/src/Icons/svgs/document.svg -- 0.54kb -> 0.53kb (0.55%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

tanmoyAtb committed:
Report modal styles and cut off fix (#1766)

* report modal fix

* removed  copy

Co-authored-by: Ryan Noble <ryanjnoble@gmail.com>

FSM1 committed:
Fix Mobile Nav Drawer blocks breadcrumb (#1768)

tanmoyAtb committed:
secondary colors update (#1769)

Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

tanmoyAtb committed:
File info modal CID and Key copy arrangements  (#1773)

* adding flags

* info modal copy buttons ready

* lingui extract

Co-authored-by: GitHub Actions <actions@github.com>

FSM1 committed:
fix share transfer modal alignment (#1774)

* fix alignment

* lint :)

Co-authored-by: Ryan Noble <ryanjnoble@gmail.com>

asnaith committed:
move pr template to .github root dir (#1776)

Co-authored-by: Ryan Noble <ryanjnoble@gmail.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

tanmoyAtb committed:
UI fix (#1777)

FSM1 committed:
resolve double tap (#1778)

Co-authored-by: Ryan Noble <ryanjnoble@gmail.com>

RyRy79261 committed:
Mobile sharing folder rename modal (#1779)

* Done#

* Update packages/files-ui/src/Components/Modules/FileBrowsers/views/FileSystemItem/SharedFolderRow.tsx

Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

* lingui extract

Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>
Co-authored-by: Tanmoy Basak Anjan <tanmoy3399@gmail.com>
Co-authored-by: GitHub Actions <actions@github.com>

RyRy79261 committed:
Rename with split extension (#1760)

* Filesplitting done

* Desktop done

* Mobile done

* Yolo files

* Fixed rename for folder

* Removing end of string

* Apply suggestions from code review

* fix lint

Co-authored-by: Tanmoy Basak Anjan <tanmoy3399@gmail.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>
Co-authored-by: Michael Yankelev <myankelev@gmail.com>

asnaith committed:
Add ui test coverage for link creation / deletion (#1775)

weblate committed:
Translated using Weblate (French) (#1784)

Currently translated at 100.0% (308 of 308 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/fr/

Co-authored-by: J. Lavoie <j.lavoie@net-c.ca>

Tbaut committed:
Bump Cypress to 9.1 (#1785)

* bump

* devdep

* lingui extract

Co-authored-by: GitHub Actions <actions@github.com>

FSM1 committed:
removing gaming dashboard from  monorepo (#1790)

* removing gaming dashboard from  monorepo

* remove leftover gaming

Tbaut committed:
Fix 0 randomly appearing (#1789)

* no 0 anymore

* more succint

* lingui extract

Co-authored-by: GitHub Actions <actions@github.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

Tbaut committed:
Fix typing issues that appeared with cypress v9.1 upgrade (#1791)

* no 0 anymore

* more succint

* lingui extract

* fix typing and definitions

* unneeded index.d.ts

* undo other commit

* unneeded void

* Use a new random user for every test run (#1792)

* new user every time

* banner test with old test user

Co-authored-by: GitHub Actions <actions@github.com>
Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>

Tbaut committed:
Edit PR template (#1798)

* template edit

* opinions :)

RyRy79261 committed:
Heic loader (#1765)

* Colors fixed

* Colors fixed

* Forced default

* Changed to transparent

* lingui extract

* resolve dark mode spinner

Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>
Co-authored-by: Tanmoy Basak Anjan <tanmoy3399@gmail.com>
Co-authored-by: Thibaut Sardan <33178835+Tbaut@users.noreply.github.com>
Co-authored-by: GitHub Actions <actions@github.com>
Co-authored-by: Michael Yankelev <myankelev@gmail.com>

Tbaut committed:
change for create (#1797)

* change for create

* lingui extract

Co-authored-by: GitHub Actions <actions@github.com>

FSM1 committed:
attempt fixing storybook build (#1801)

* attempt fixing storybook build

* actually fix the error

Co-authored-by: Thibaut Sardan <33178835+Tbaut@users.noreply.github.com>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ChainSafe/ui-monorepo?pullRequest=1806) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/ChainSafe/ui-monorepo?pullRequest=1806) before we can accept your contribution.<br/>**5** out of **7** committers have signed the CLA.<br/><br/>:white_check_mark: asnaith<br/>:white_check_mark: RyRy79261<br/>:white_check_mark: FSM1<br/>:white_check_mark: Tbaut<br/>:white_check_mark: tanmoyAtb<br/>:x: dependabot[bot]<br/>:x: imgbot[bot]<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ChainSafe/ui-monorepo?pullRequest=1806) it.</sub>

FSM1 committed:
add sorting header for mobile file browser (#1786)

* add sorting header for mobile file browser

* lingui extract

* fix lint

* add translations

* update sorting

* lingui extract

* remove unused classes

* remove unnecessary code

Co-authored-by: GitHub Actions <actions@github.com>
Co-authored-by: Thibaut Sardan <33178835+Tbaut@users.noreply.github.com>

FSM1 committed:
Add header content for selected (#1796)

* add sorting header for mobile file browser

* lingui extract

* fix lint

* add translations

* Add selection header

* fix lint

* Update packages/files-ui/src/Components/Modules/FileBrowsers/views/FilesList.tsx

Co-authored-by: Thibaut Sardan <33178835+Tbaut@users.noreply.github.com>

* fix toggle all call

* add indeterminate checkbox

* Update packages/files-ui/src/Components/Modules/FileBrowsers/views/FilesList.tsx

Co-authored-by: Thibaut Sardan <33178835+Tbaut@users.noreply.github.com>

Co-authored-by: GitHub Actions <actions@github.com>
Co-authored-by: Thibaut Sardan <33178835+Tbaut@users.noreply.github.com>

weblate committed:
Translated using Weblate (French) (#1811)

Currently translated at 100.0% (309 of 309 strings)

Translation: ChainSafe Files/Chainsafe Files user interface
Translate-URL: https://hosted.weblate.org/projects/chainsafe-files/chainsafe-files-user-interface/fr/

Co-authored-by: J. Lavoie <j.lavoie@net-c.ca>

FSM1 committed:
fix breaking tests (#1813)

tanmoyAtb reviewed (approved):
🚀

Tbaut committed:
Enforce props on a new line for multiline components (#1807)

* lint

* lingui extract

Co-authored-by: Michael Yankelev <12774278+FSM1@users.noreply.github.com>
Co-authored-by: GitHub Actions <actions@github.com>
Co-authored-by: Tanmoy Basak Anjan <tanmoy3399@gmail.com>
